code/modules/client/preference_setup/loadout/loadout_factions.dm

```diff
@@ -87,7 +87,7 @@
 	idris_labcoats["idris labcoat, alt"] = /obj/item/clothing/suit/storage/toggle/labcoat/idris/alt
 	idris_labcoats["idris letterman labcoat"] = /obj/item/clothing/suit/storage/toggle/labcoat/idris/letterman
 	idris_labcoats["idris letterman labcoat, alt"] = /obj/item/clothing/suit/storage/toggle/labcoat/idris/letterman/alt
-	idris_labcoats["idris labcoat, long"] = /obj/item/clothing/suit/storage/toggle/longcoat/idris
+	idris_labcoats["idris labcoat, long"] = /obj/item/clothing/suit/storage/toggle/labcoat/longcoat/idris
```

Comment:
consider `long` instead of `longcoat`.
(comment applies to below applicable entries as well.)